content/02.introduction.md

```diff
@@ -22,29 +22,64 @@ As researchers worldwide work to characterize SARS-CoV-2 and COVID-19, informati
 This is a real-time, collaborative effort that welcomes submissions from scientists worldwide.
 
-### Coronaviruses: What are they, and what do we know about SARS-CoV-19?
-
-Coronaviruses are RNA viruses that...
-[Summarize relevant mechanisms for cell entry & address evidence for/against ACE2 being important]
-
+### About Coronaviruses
+
+#### Classification:
+Coronaviridae, commonly called Coronavirus, is a family of positive-sense RNA viruses. 
+Members of this family include the Middle East respiratory syndrome-related coronavirus (MERS-CoV) and Severe acute respiratory syndrome coronavirus (SARS-CoV), which are two species, as well as SARS-CoV-2.
+
+#### Virion, morphology, structure:
+Coronavirus virions are spherical with diameters ranging between 100 to 160nm. 
+The virion has a lipid envelope with spike (S) glycoproteins on the surface creating a distinctive "crown" shape, for which the family of viruses was named. 
+
+#### Genome structure and replication:
+Coronavirus genomes are single-stranded non-segmented positive RNA (ssRNA+) that are 27 to 32kb in length. 
+The SARS-CoV-2 genome lies in the middle of this range at 29,903 bp in length [@doi:10.1016/S0140-6736(20)30251-8].
+Coronavirus genomes are comprised of a replicase gene that is 2/3 of the genome and the remaining 1/3 that encodes structural proteins including: spike (S), membrane (M), envelope (E), and nucleocapsid (N) proteins. 
+There are additional accessory genes that are present depending on the strain. 
+The non-structural polypeptide is translated into 16 non-structural proteins (nsp1-16, except Gammacoronaviruses that do not contain nsp1), which form replication machinery to synthesize viral RNA. 
+
+Replication initiates with the binding of the virus to a host cell via the S protein. 
+The viral membrane fuses with the endosomal membrane to release the viral genome into the host cytoplasm. 
+The replicase gene is translated and assembled into the viral replicase complex, which can synthesize dsRNA genome from the genomic ssRNA(+).
+The dsRNA genome is then transcribed and replicated to create viral mRNAs and new ssRNA(+) genomes.  
+
+
+#### Transmission:
+In general, respiratory viruses like coronavirus can have multiple routes of person-to-person transmission including droplet transmission (i.e. inhalation for cough, sneeze), aerosol transmission (i.e. virus suspended in air), and contact transmission (i.e. contact with oral, nasal, and eye mucous membranes). 
+While there does not appear to be experimental evidence to conclude which mode of transmission different coronaviruses use [@doi:10.1038/s41368-020-0075-9], it is suggested that the coronavirus person-to-person transmission is droplet-based using simulations [@doi:10.1056/NEJMoa032867] and trace studies[@doi:10.3390/ijerph15122757].
+
+References:
```

Comment:
I didn't notice these! Is there somewhere they could go inline?

Reply:
yes I'll update this